### PR TITLE
when items have no release tags, the update-marc step should still be…

### PR DESCRIPTION
**Pull request by peetucket**
… able to run without erroring out

this bug fix has already been applied to the new way of doing this with dor-services-app: https://github.com/sul-dlss/dor-services-app/pull/31

this is just a way of getting the bug fix out to production now

### CODE REVIEW

File: lib/dor/update_marc_record_service.rb [hunk 1]
```diff
@@ -172,7 +172,7 @@ def get_x2_constituent_info
 
     def released_to_Searchworks
       rel = @druid_obj.released_for
-      rel['Searchworks']['release']
+      rel.blank? || rel['Searchworks'].blank? || rel['Searchworks']['release'].blank? ? false : rel['Searchworks']['release']
```
Comment (mejackreed) [inline lib/dor/update_marc_record_service.rb]: I'm curious about the string `Searchworks` here. Where is that coming from?

Currently in prod we have 3 or 4 variations of this, and I setup Argo to do something different. Can we figure out what this should be or have it configurable?

Comment (peetucket) [inline lib/dor/update_marc_record_service.rb]: All good questions that need to be discussed with the team.

Comment (lmcglohon) [inline lib/dor/update_marc_record_service.rb]: @mejackreed @peetucket Was this discussed last week?  What was the outcome?

Comment (peetucket) [inline lib/dor/update_marc_record_service.rb]: for now, we decided to just make the tags case insensitive --- but the searchworks string itself will still be hardcoded for now

sorry, i forgot to mark this pull request as unnecessary .... if we can merge and deploy the updated dor-services-app and item-release robots, this logic will be gone from here (and be in dor-services-app)

the latest version of this code in dor-services-app makes the whole release tag case insensitive

**Comment by peetucket** [comment]
I'm just going to close this pull request...it is not needed now that we have move functionality over to dor-services-app

**Comment by lmcglohon** [comment]
@peetucket Can we delete the branch as well or would you like to keep it for now?

**Comment by peetucket** [comment]
i deleted

**Pull request closed without merging**